app/react/src/server/config/webpack.config.prod.js

```diff
@@ -29,7 +29,7 @@ export default function() {
           screw_ie8: true,
           warnings: false,
         },
-        mangle: false,
+        mangle: true,
```

Comment:
Because it is renaming symbols. If there is any sort of metaprogramming in the code, e.g. any use of eval or even non-static access to members, let's say `object['x' + Math.random()]` then it will break. Storybook can't/shouldn't make these assumptions about third-party code.